google-cloud-spanner/src/test/java/com/google/cloud/spanner/BatchClientImplTest.java

```diff
@@ -102,6 +102,7 @@ public void setUp() {
     @SuppressWarnings("resource")
     SpannerImpl spanner = new SpannerImpl(gapicRpc, spannerOptions);
     client = new BatchClientImpl(spanner.getSessionClient(db), isMultiplexedSession);
+    BatchClientImpl.unimplementedForPartitionedOps.set(false);
```

Comment:
This by default would have been set to `false` right? Do we still need this?
https://github.com/googleapis/java-spanner/blob/c12968a5f6dad95017d9867d96d4f19a26643a07/google-cloud-spanner/src/main/java/com/google/cloud/spanner/BatchClientImpl.java#L69

Reply:
Since unimplementedForPartitionedOps is a static variable, running complete test suite creates flaky tests.